x509/revocation/mozilla/mozilla.go

```diff
@@ -72,8 +74,9 @@ type Entry struct {
 
 // SubjectAndPublicKey specifies a revocation entry by Subject and PubKeyHash
 type SubjectAndPublicKey struct {
```

Comment:
Is this going to cause issues for users, since we're heavily changing an exported strucy?

Reply:
yes, if they change the commit.
but the existing users are already broken or not using the latest OneCRL as the schema has changed and the current implementation fails to parse the most recent CRL set.